packages/terra-title/examples/Index.jsx

```diff
@@ -0,0 +1,149 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import Markdown from '../../../packages/terra-site/src/Markdown';
+import ReadMe from '../docs/README.md';
+import '../src/Title.scss';
+
+
+const TitleExamples = () => (
+  <div>
+    <Markdown id="readme" src={ReadMe} />
+    <h2 id="caption">Caption</h2>
+    <h1 className="terra-Title terra-Title--tiny terra-Title--horizontalMiddle">
+      <span className="terra-Title--heading">
+        Terra Title Horizontal Middle Caption
+        <div className="terra-Title--caption">Terra Title Caption</div>
+      </span>
+    </h1>
+    <br />
+    <h1 className="terra-Title terra-Title--tiny terra-Title--verticalCenter">
+      <span className="terra-Title--heading">
+        Terra Title Vertical Center Caption
+        <div className="terra-Title--caption">Terra Title Caption</div>
+      </span>
+    </h1>
+    <br />
+    <br />
+    <h2 id="graphic">Graphic</h2>
+    <h1 className="terra-Title terra-Title--tiny terra-Title--horizontalMiddle">
+      <span className="terra-Title--graphic">
+        <i className="fa fa-cloud" />
```

Comment:
Ah, good question. We don't have font-awesome as a dev-dependency, [we pull in the font-awesome icons via a link tag to a CSS file for font-awesome on a cdn](https://github.com/cerner/terra-ui/pull/91/commits/39a63b9e2b5a9c804cc5e2f1cee96d8f779f4c4d).

This is only temporary, once terra-icon lands in the monorepo and we update this to a react component, we'll stop referencing font-awesome icons and switch to terra-icon component.